controllers/openstackdataplane_controller.go

```diff
@@ -226,6 +227,7 @@ func (r *OpenStackDataPlaneReconciler) SetupWithManager(mgr ctrl.Manager) error
 		For(&dataplanev1beta1.OpenStackDataPlane{}).
 		Owns(&dataplanev1beta1.OpenStackDataPlaneNode{}).
 		Owns(&dataplanev1beta1.OpenStackDataPlaneRole{}).
+		WithOptions(controller.Options{MaxConcurrentReconciles: 2}).
```

Comment:
I believe we need 2 for dataplane and dataplanerole reconciles

Reply:
I believe that even if it set to 1 it should queue up the reconcile request and execute them sequentially.

Reply:
I'm not sure about this. I don't see why we would want >1 reconciles.
FWIW, I'm not seeing the deploy not starting issue when I test with https://github.com/openstack-k8s-operators/dataplane-operator/pull/120 locally.

Reply:
I'll close it then